File: help/eng_gb/exercicios_da_ficha_4/Ficha04_017/Ficha04_017.js
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Grade \t: " );// print text
    grade = parseInt(prompt("Grade \t: " ,"0") );
    
    document.write(grade + "\n");// print input 
    if(grade < 0 || grade > 20 ){
        document.write("Invalid Grade" );
    }else{
        document.write("Valid Grade" );
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:29

File: help/eng_gb/exercicios_da_ficha_4/Ficha04_017/Ficha04_017.c
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Grade \t: ");
    int grade;
    scanf("%d", &grade);
    if(grade < 0 || grade > 20 ){
        printf("Invalid Grade");
    }else{
        printf("Valid Grade");
    }
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:29
